AsyncDisplayKit/ASImageNode+AnimatedImage.h

```diff
@@ -0,0 +1,23 @@
+//
+//  ASImageNode+AnimatedImage.h
+//  Pods
```

Comment:
Probably shouldn't say Pods - copy another header, feel free to keep your name and date :)